api/preprint_providers/serializers.py

```diff
@@ -143,3 +147,69 @@ def update(self, instance, validated_data):
         instance.reviews_comments_anonymous = validated_data['reviews_comments_anonymous']
         instance.save()
         return instance
+
+class ModeratorSerializer(JSONAPISerializer):
+    filterable_fields = frozenset([
+        'full_name',
+        'id',
+        'permission_group'
+    ])
+
+    id = IDField(source='_id', required=True)
+    full_name = ser.CharField(source='fullname', required=False, label='Full name', help_text='Display name used in the general user interface', max_length=186)
+    permission_group = ser.CharField(required=True)
+
+    class Meta:
+        type_ = 'moderators'
+
+    def get_absolute_url(self, obj):
+        return absolute_reverse('moderators:provider-moderator-detail', kwargs={
+            'provider_id': self.context['request'].parser_context['kwargs']['version'],
+            'moderator_id': obj._id,
+            'version': self.context['request'].parser_context['kwargs']['version']})
+
+    def create(self, validated_data):
```

Comment:
Couple "not big deal" thoughts on `create`:
1. user_id being id or user guid is a little funny, but not a huge deal.

Reply:
It's an email or guid, not a pk

Reply:
Er yeah, I should proofread. I swear email or guid is what I meant.

Reply:
I agree, is unusual. The alternative is having a write-only email field and making neither required.